### PR TITLE
Fix some grammatical mistakes

### PR DESCRIPTION
**Pull request by gxyd**

### CODE REVIEW

File: docs/aboutus.rst [hunk 1]
```diff
@@ -475,22 +475,22 @@ competitive and fast (from 10x to 100x) by rewriting the core from scratch.
 Jurjen N.E. Bos has contributed pretty printing and other patches.
 Fredrik Johansson has wrote mpmath and contributed a lot of patches.
 
-SymPy has participated in every GSoC since 2007.  Moderate amount
+SymPy has participated in every GSoC since 2007. Moderate amount
```
Comment (skirpichev) [inline docs/aboutus.rst]: Two spaces after the sentence - is not a grammar error.  This is a "Emacs thing" and more or less - convention for open source world, part of GNU coding standard.

File: docs/aboutus.rst [hunk 2]
```diff
@@ -475,22 +475,22 @@ competitive and fast (from 10x to 100x) by rewriting the core from scratch.
...
 also started as a GSoC student, taking his place.
 
 Ondřej Čertík is still active in the community, but is too busy with work
-and family to play a lead development role.  Unfortunately, his remaining
+and family to play a lead development role. Unfortunately, his remaining
 activity neither constructive nor productive anymore and SymPy just
-slowly dying now: most former contributors are inactive now or
-explicitely leave this "friendly and welcoming" project.
+slowly dies now: most former contributors are inactive now or
+explicitly leaving this "friendly and welcoming" project.
```
Comment (skirpichev) [inline docs/aboutus.rst]: Ok.  Thank you.

**Comment by gxyd** [comment]
I have made the required changes. I could use "git rebase -i" to merge commits into single one if you prefer, since the last two commits make the same changes. I am not aware of the git policy towards that in the project.

**Comment by skirpichev** [comment]
> I could use "git rebase -i" to merge commits into single one if you prefer

I believe, that this should be doable from the github interface (on merge).

So, don't do this.

**Comment by skirpichev** [comment]
OK, this is in.

Let me know if you see anything else to fix.  But next time please don't push every commit to Github.  This will trigger multiple CI builds.  Make some logical piece of your work locally, then publish all your commits in a single shot.

**Pull request merged**